src/Admin/BaseFieldDescription.php

```diff
@@ -475,13 +475,22 @@ public function setHelp($help)
         $this->help = $help;
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/admin-bundle 3.x and will be removed in version 4.0. Use Symfony Form "help" option instead.
```

Comment:
The version where this method was deprecated is already released, so "3.74" should be used instead of "3.x".